.github/workflows/main.yml

```diff
@@ -157,6 +170,11 @@ jobs:
       SHELL: "/bin/bash" # To prevent https://github.com/kubernetes/minikube/issues/6643
     needs: [build_minikube]
     steps:
+    - name: Install kubectl
+      shell: bash
+      run: |
+        curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/amd64/kubectl
+        sudo install kubectl /usr/local/bin/kubectl
```

Comment:
do you need to make it executable too ?

https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux

Reply:
I think install makes it executable. doesnt it ?

Reply:
Ah, right. They used `mv` or `cp`. Maybe install fixes the owner, too